lib/indent.js

```diff
@@ -25,21 +23,22 @@ var _specialTypes = {
 exports.setOptions = setOptions;
 function setOptions(opts){
   _opts = opts;
+  indent.setOptions(opts);
 }
 
 
 // transform AST in place
 exports.transform = transform;
 function transform(ast) {
-  rocambole.moonwalk(ast, transformNode);
```

Comment:
starting from root nodes to allow negative indent levels, and because of that ended up needing to replace many of the `edges`...